### PR TITLE
Add some emoji flare to the current AQI

### PR DESCRIPTION
**Pull request by obra**
I'm not wedded to the specific emoji. Heck, I won't be offended if the idea of sticking faces on the AQI is rejected outright.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -58,7 +58,7 @@
     const distance = Math.round(closestSensor.distance * 10) / 10
     const time = (new Date()).toLocaleTimeString()
     const paLink = getPurpleAirLink()
-    const aqiMsg = `AQI is ${aqi}`
+    const aqiMsg = `${getAQIEmoji(aqi)} AQI is ${aqi}`
```
Comment (skalnik) [inline app.js]:
```suggestion
    const aqiMsg = `AQI is ${aqi} ${getAQIEmoji(aqi)}`
```

I don't know why but emoji as endings read better to me than having them lead. What do you think?

File: app.js [hunk 2]
```diff
@@ -190,6 +190,27 @@
...
+    } else if (aqi >= 151) {
+      return '&#x1F637;'; // 😷
+    } else if (aqi >= 101) {
+      return '&#x1F641;'; // ☹️
+    } else if (aqi >= 51) {
+      return '&#x1F610;'; // 😐
+    } else if (aqi >= 0) {
+      return '&#x1F600';  // 😀
+    } else {
+      return undefined;
```
Comment (skalnik) [inline app.js]:
```suggestion
      return "";
```

I think we should just return empty string in the worst case. It's unlikely to happen, but there's no upside to putting an `undefined` string in the text we present.

**Review by skalnik — COMMENTED** [review]
Hah! I love it. I added a few suggestions.

**Comment by obra** [comment]
Both of those suggestions look good. Committed.

**Pull request merged**